pynopegl/setup.py

```diff
@@ -106,6 +106,42 @@ def generate(self):
         livectl_info = self._livectl_info()
         return classes + "\n\n" + livectl_info
 
+    @classmethod
+    def _get_combined_type(cls, node_types, wrap_into_union=True):
+        if not node_types:
+            return "Node"
+
+        # We're escaping the node name because we're the node are not declared
```

Comment:
`because we're the node are not declared`, typo ?

Reply:
Oups yeah, reworded into:
```python
# We're escaping the node name into quotes because the node classes are
# not declared according to their cross dependencies
```